workers/ohsome_quality_analyst/utils/helper.py

```diff
@@ -167,3 +169,17 @@ def flatten_sequence(input_seq: Union[dict, list, tuple, set]) -> list:
         else:
             output.append(val)
     return output
+
+
+def load_sklearn_model(path: str):
+    """Load sklearn model from disk
+
+    Raise an error if during load back of a pickled model a UserWarning is produced.
```

Comment:
I'm not sure if joblib produces pickled models. We only support models that have been dumped with joblib, right?

Reply:
Right, joblib dumps are like pickles. But to avoid confusion I corrected the docstring.